### PR TITLE
Add code block surround operator for org-mode

### PR DESCRIPTION
**Pull request by TheBB**

### CODE REVIEW

File: layers/+emacs/org/funcs.el [hunk 1]
```diff
@@ -0,0 +1,18 @@
+;;; packages.el --- rebox layer function file for Spacemacs.
```
Comment (nixmaniack) [inline layers/+emacs/org/funcs.el]: org layer function file for Spacemacs.

Comment (TheBB) [inline layers/+emacs/org/funcs.el]: Of course!

File: layers/+emacs/org/packages.el [hunk 2]
```diff
@@ -57,11 +57,9 @@
...
 (defun org/post-init-evil-surround ()
   (defun spacemacs/add-org-surrounds ()
-    (push '(?: . spacemacs//surround-drawer) evil-surround-pairs-alist))
-  (add-hook 'org-mode-hook 'spacemacs/add-org-surrounds)
-  (defun spacemacs//surround-drawer ()
-    (let ((dname (read-from-minibuffer "" "")))
-      (cons (format ":%s:\n" (or dname "")) "\n:END:"))))
+    (push '(?: . spacemacs//surround-drawer) evil-surround-pairs-alist)
+    (push '(?# . spacemacs//surround-code) evil-surround-pairs-alist))
+  (add-hook 'org-mode-hook 'spacemacs/add-org-surrounds))
```
Comment (syl20bnr) [inline layers/+emacs/org/packages.el]: Do we have to do this in a hook or could it be done only once with a `spacemacs|add-use-package-hook` ?

Comment (TheBB) [inline layers/+emacs/org/packages.el]: I think it has to be in a hook because `evil-surround-pairs-alist` must be buffer-local.

**Comment by TheBB** [comment]
Merged.

**Pull request closed without merging**